src/Auditable.php

```diff
@@ -143,6 +143,13 @@ protected function resolveAuditExclusions()
      */
     public function getAuditExclude(): array
     {
+        if ( 
+            Config::get('audit.exclude_merge', false) || 
+            ($this->auditExcludeMerge ?? false) 
+        ) {
```

Comment:
https://github.com/owen-it/laravel-auditing/pull/967#discussion_r1765671881
It seems that you decided not to follow the instructions
```php
if ($this->auditExcludeMerge ?? Config::get('audit.exclude_merge')) {
```
The specific of the model must replace the global, 
IMO, if i set `$auditExcludeMerge = false;`, It is expected that the merge will not be done even though the global `'audit.exclude_merge' => true`

Reply:
I do not see how the code is different, but I updated it. Functionally it seems to be the same. But I updated it.

Reply:
Not the same, 
enjoy: [https://onlinephp.io?s=s7EvyCj](https://onlinephp.io?s=s7EvyCjg5dLXV3DNK0ktUqjMLy1SSM5PSVXISC1K1VFIzcvKr1Tk5VJJz8lPSsxxLE3JLHGtSM4pTUn1TS1KT1WwVSgpKk21BqpIxCKXlphTDJIsSyyKTynNLdDAosreXgGX6ZpArSBIpu15pTk5NLUc5r1BaDs0XMi2HQA%2C&v=8.2.20)

Reply:
Perhaps this:
```
 if (
            (Config::get('audit.exclude_merge', false) && ($this->auditExcludeMerge ?? false) !== false))
        ) {
            return merge_array($this->auditExclude ?? [], Config::get('audit.exclude', []);
        }
```